doc/flame/components.md

```diff
@@ -788,6 +789,14 @@ This is an example of how a quarter-length map looks like:
 
 ![An example of a isometric map with selector](../images/isometric.png)
 
+The component Also supports negative indexing starting from -2 *(as -1 is reserved for empty tile)* which returns sprite from bottom Left to bottom right using negative Indexing and wraps to upper rows on overflow of tileset's image width. 
+
+Use `propSize` Property to give a custom Size to negative Indexed elements *(This can be used to add props and art as Tiles which rise higher than most tiles)*.
```

Comment:
```suggestion
Use the `propSize` property to give a custom size to negative indexed elements 
*(This can be used to add props and art such as tiles which rise higher than most tiles)*.
```